res/res_pjsip_config_wizard.c

```diff
@@ -1043,6 +1049,19 @@ struct ast_sorcery_instance_observer observer = {
 	.object_type_loaded = object_type_loaded_observer,
 };
 
+
+/*! \brief Callback for Named ACL changed */
+static void acl_change_stasis_cb(void *data, struct stasis_subscription *sub, struct stasis_message *message)
+{
+	/* We will verify whether it is an "ACL Change" message. */
+	if (stasis_message_type(message) != ast_named_acl_change_type()) {
+		return;
+	}
+
+	ast_debug(3, "PJSIP Wizard: Named ACL change detected via Stasis.\n");
+	acl_change_detected = 1;
```

Comment:
The commit message says this forces a reload on named ACL changes, but this code doesn't. It should and then reset the ACL change detected variable after.